test/development/acceptance-app/ReactRefreshLogBox-builtins.test.ts

```diff
@@ -150,29 +150,29 @@ describe.each(['default', 'turbo'])('ReactRefreshLogBox app %s', () => {
     const source = await session.getRedboxSource()
     if (process.env.TURBOPACK) {
       expect(source).toMatchInlineSnapshot(`
-        "./app/page.js:2:1
-        Module not found: Can't resolve 'b'
-          1 | 'use client'
-        > 2 | import Comp from 'b'
-            | ^^^^^^^^^^^^^^^^^^^^
-          3 | export default function Oops() {
-          4 |   return (
-          5 |     <div>
-
-        https://nextjs.org/docs/messages/module-not-found"
+       "./app/page.js (2:1)
```

Comment:
follow-up: Why are these in round brackets? This seems like a pretty exotic variation.